files-csharp/FastestWayToReadATextFileInCsharp/Tests/WaysToReadATextFileInCsharpTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using FastestWayToReadATextFileInCsharp;
+
+namespace Tests
+{
+    public class WaysToReadATextFileInCsharpTests
+    {
+        private static readonly string _sampleFilePath
+            = Path.Combine(Directory.GetParent(Directory.GetCurrentDirectory()).Parent.Parent.FullName, "SampleTextFile.txt");
+
+        private readonly WaysToReadATextFileInCsharp _textFileReader;
+
+        public WaysToReadATextFileInCsharpTests()
+        {
+            _textFileReader = new WaysToReadATextFileInCsharp(_sampleFilePath);
+        }
+
+        private readonly string _expected
```

Comment:
This can be made constant

Reply:
As an additional thought, since this is also the expected text from the sample file. Rather than having the sample file, we could simply write this text to a temp file and then use that temp file in the tests (being sure to clean up the temp file when we are done). This would also make the tests more robust as we would not be facing the issue of having to keep both the file and this string in synch.